src/testUtils/TestRig.react.js

```diff
@@ -57,10 +57,17 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
-  toggleCheckbox(selector) {
-    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
-    $(element).prop('checked', !($(element)[0].checked))
-    TestUtils.Simulate.change(element[0])
+  toggleCheckbox(element) {
+    element.checked = !(element.checked)
+    TestUtils.Simulate.change(element)
+  }
+  toggleRadioButton(element) {
+    element.checked = !(element.checked)
+    TestUtils.Simulate.change(element)
+  }
+  select(element, value) {
```

Comment:
@bartriordan In our app the value of radio button is either "enabled" or ""disabled", so we can't set value to boolean here, that's why I'm using the checked property